README.md

```diff
@@ -1,49 +1,211 @@
 # TechDocs CLI
 
-Check out the [TechDocs README](https://github.com/backstage/backstage/blob/master/plugins/techdocs/README.md) to learn more.
+[![](https://img.shields.io/npm/v/@techdocs/cli)](https://www.npmjs.com/package/@techdocs/cli)
 
-**WIP: This cli is a work in progress. It is not ready for use yet. Follow our progress on [the Backstage Discord](https://discord.gg/MUpMjP2) under #docs-like-code or on [our GitHub Milestone](https://github.com/backstage/backstage/milestone/15).**
+Utility command line interface for managing TechDocs sites in [Backstage](https://github.com/backstage/backstage).
 
-## Prerequisities
+https://backstage.io/docs/features/techdocs/techdocs-overview
 
-Install the techdocs-cli:
+## Features
+
+- Supports local development/preview of a TechDocs site in a Backstage app.
+- Supports generation and publishing of a documentation site in a CI/CD workflow.
+
+```
+$ techdocs-cli --help
+Usage: techdocs-cli [options] [command]
+
+Options:
+  -V, --version             output the version number
+  -h, --help                display help for command
+
+Commands:
+  generate|build [options]  Generate TechDocs documentation site using mkdocs.
+  publish [options]         Publish generated TechDocs site to an external storage AWS S3,
+                            Google GCS, etc.
+  serve:mkdocs [options]    Serve a documentation project locally using mkdocs serve.
+  serve [options]           Serve a documentation project locally in a Backstage app-like
+                            environment
+  help [command]            display help for command
+```
+
+## Installation
+
+You can always use [`npx`](https://github.com/npm/npx) to run the latest version of `techdocs-cli` -
+
+```bash
+$ npx @techdocs/cli [command]
```

Comment:
nit: I prefer not prefixing with `$` and making the code fence `bash` or `sh`. It's easier to copy the example usage. Also more consistent with lines below.

Reply:
That's a good point.

- [ ] Will update.